src/PkId/PkId.php

```diff
@@ -1,31 +1,21 @@
 <?php
 
+declare(strict_types=1);
+
 namespace PK\PkId;
 
-/**
- * Class PkId
- * @package PK\PkId
- */
 class PkId
 {
-    /**
-     * @var int
-     */
-    private $id;
+    /** @phpstan-ignore-next-line */
+    private int $id;
 
-    /**
-     * @param int $id
-     */
     public function __construct(int $id)
     {
         $this->id = $id;
     }
 
-    /**
-     * @return string
-     */
     public function getId(): string
     {
-        return md5(rand(0, 1000000));
+        return md5((string)rand(0, 1000000));
```

Comment:
propozycja żeby zwiększyć 1000000 na 2000000 bo są komputery kwantowe i to stare może być za słabe